crates/admin/src/cluster_controller/cluster_state.rs

```diff
@@ -145,19 +145,26 @@ impl<T: TransportConnect> ClusterStateRefresher<T> {
                     .name("get-processors-state")
                     .spawn(async move {
                         tc.run_in_scope("get-processor-state", None, async move {
-                            (
-                                node_id,
-                                tokio::time::timeout(
-                                    // todo: make configurable
-                                    std::time::Duration::from_secs(1),
-                                    rpc_router.call(
-                                        &network_sender,
+                            match network_sender.node_connection(node_id).await {
```

Comment:
I believe I don't need to cache this since it will either get a random existing connection, or attempt to connect afresh.

Reply:
Agreed